### PR TITLE
Add alert info & rtl style & favicon

### PR DESCRIPTION
**Pull request by dever404**
@soufyanelf

### CODE REVIEW

**Comment by soufyanelf** [comment]
please resolve the conflict

**Comment by soufyanelf** [comment]
@Everkers are you ok if I merge this?

File: src/components/MuiProvider/index.js [hunk 1]
```diff
@@ -54,12 +57,17 @@ export const muiTheme = createTheme({
...
+  key: 'muirtl',
+  stylisPlugins: [prefixer, rtlPlugin],
+});
 
 // MaterialUI Theme Provider only runs on the client side, so we need to wrap it in a component that only runs on the client side
 export const MuiProvider = ({ children }) => {
   return (
     <ThemeProvider theme={responsiveFontSizes(muiTheme)}>
-      {children}
+      <CacheProvider  value={cacheRtl}>{children}</CacheProvider>
```
Comment (Everkers) [inline src/components/MuiProvider/index.js:70]: image

This bug occurs when using the cacheProvider

File: src/components/Header/index.jsx [hunk 2]
```diff
@@ -3,10 +3,12 @@ import Link from "next/link";
 import Image from "next/image";
 import styles from "./header.module.css";
 import { Button, Container, Typography } from "@mui/material";
+import  TransitionAlerts from "../Alert";
 
 const Header = ({ navComponent }) => {
   return (
     <>
+      <TransitionAlerts></TransitionAlerts>
```
Comment (Everkers) [inline src/components/Header/index.jsx:11]: Use a self closing tag

**Review by Everkers — CHANGES_REQUESTED** [review]

File: src/components/Header/index.jsx (hunk 2 above)
Comment (dever404) [reply src/components/Header/index.jsx]: [3908d19](https://github.com/BizTech-Morocco/mosa3ada.biztech.ma/pull/33/commits/3908d1945669a544c23b0d6a5beeb0ec9df15320)

Comment (Everkers) [reply src/components/Header/index.jsx]: image
Still getting the same bug

Comment (Everkers) [reply src/components/Header/index.jsx]: If you're only utilizing the CacheProvider just for RTL support, there are alternative workarounds to get same results

**Pull request closed without merging**